test/e2e/scaffold/apisix.go

```diff
@@ -72,7 +72,7 @@ spec:
             tcpSocket:
               port: 9080
             timeoutSeconds: 2
-          image: "localhost:5000/apache/apisix:dev"
+          image: "localhost:5000/apisix:dev"
```

Comment:
We could also add a global variable that stores the $REGISTRY used instead of hardcoding it here.

Reply:
Relevant processing logic already exists.
https://github.com/apache/apisix-ingress-controller/blob/master/test/e2e/scaffold/scaffold.go#L588